### ISSUE TITLE
about the validation data split

### ISSUE THREAD

**Issue by JAYatBUAA** [fnzhan/RABIT#5]
Dear author,
Sorry, I'm a beginner in this field. To my knowledge, data will be splited into train, validation and test set, and we can calculate the FID score on validation data for model selection during training. However, I didn't find the validation set split and related code in the RABIT, as well as CoCosNet, which makes me confused. Thanks in advance.
